src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -34,57 +34,27 @@ class ScalaPBGenerator extends Processor {
   def deleteDir(path: Path): Unit =
     try DeleteRecursively.run(path)
     catch {
-      case e: Exception => ()
+      case _: Exception => ()
```

Comment:
I wonder if we should at least log here. I wonder if deletes failing could be part of the problem with our rare reproducibility issues (we have old class files laying around, and bazel reuses paths).

Reply:
done, didn't print the stack trace since felt it will be too verbose, added explanation and exception message